service/src/main/java/greencity/service/ubs/EventServiceImpl.java

```diff
@@ -35,4 +36,37 @@ public void save(String eventName, String eventAuthor, Order order) {
         event.setOrder(order);
         eventRepository.save(event);
     }
+
+    /**
+     * This method return correct status for changes with Responsible employee.
+     *
+     * @param positionId    ID changed position.
+     * @param existedBefore If True - We update Info, otherwise we assign
+     * @author Rostyslav Sikhovskiy.
+     */
+    @Override
+    public String changesWithResponsibleEmployee(Long positionId, Boolean existedBefore) {
+        if (existedBefore.equals(Boolean.TRUE)) {
+            if (positionId == 2) {
```

Comment:
can you define these relations in map instead?

Reply:
Fixed